### DISCUSSION TITLE
Weird graphics even with Cairo installed

### DISCUSSION THREAD

**Discussion by mdanb** [GreenleafLab/ArchR#1842] [Questions / Documentation] [answered]
I'm trying to plot marker genes on my UMAP plot. However, the graphic I get is very weird (see below), even though I did install `Cairo` in my conda environment. I re-ran the code in my friend's conda environment and his version looks good (see below). We're not sure what package (missing or present in my environment) is causing the issue. 

My version (screenshot):
![Screenshot from 2023-02-09 16-41-59](https://user-images.githubusercontent.com/37456158/217945849-963d1beb-7dc3-4743-bf9c-a18ed04b900f.png)

His version:
[gene_marker_UMAPs_Tsankov_lung.pdf](https://github.com/GreenleafLab/ArchR/files/10701974/gene_marker_UMAPs_Tsankov_lung.pdf)

**Comment by rcorces** [comment, marked as answer]
https://github.com/GreenleafLab/ArchR/issues/1731
